config/routes.rb

```diff
@@ -1,9 +1,14 @@
 Rails.application.routes.draw do
   root to: "welcome#index"
 
+  as :user do
+    patch '/user/confirmation' => 'confirmations#update', :via => :patch, :as => :update_user_confirmation
+  end
```

Comment:
What does this do?

Reply:
update confirmation is something we added to set passwords upon confirmation, so we need to specify the additional route outside of the Devise one I believe.